db/migrate/20160324174132_create_container_deployments.rb

```diff
@@ -0,0 +1,24 @@
+class CreateContainerDeployments < ActiveRecord::Migration[5.0]
+  def change
+    create_table :container_deployments do |t|
+      t.string   :deployment_type
+      t.string   :deployment_version
+      t.boolean  :containerized, :default => true
+      t.string   :deployment_method
+      t.string   :ssh_private
+      t.string   :ssh_public
+      t.string   :ssh_user
+      t.string   :rhsm_user
+      t.string   :rhsm_pass
+      t.string   :rhsm_sku
+      t.string   :rhsm_pool_id
+      t.string   :rhsm_server
+      t.string   :rhsm_activation_key
```

Comment:
@alongoldboim please add `metrics_endpoint` (which will be something like `hostname:port`).

cc @detiber I know you have something similar on your side as `metrics_hostname` but I think the name should be `endpoint` as it may contain a port as well.